### ISSUE TITLE
Elevated fractional excretion of urate

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8029]
**Preferred term label:**
Elevated fractional excretion of urate

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
220150 
PMID: 12024214  1/1
PMID: 14655203   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000746
- label: Elevated fractional excretion of urate
- parents: [Abnormality of renal excretion (HP:0011036)](https://hpo.jax.org/app/browse/term/HP:0011036)
- pmid: PMID:12024214|PMID:14655203
- def: Abnormally increased proportion of filtered urate that is excreted in the urine as calculated using the concentrations of urate and creatinine in both blood and urine. Clinically this manifests as uric acid wasting.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8029
- synonyms: Uric acid wasting(NONE)
